lib/expand/join.go

```diff
@@ -848,31 +838,69 @@ func (p *Peer) validateWizardState(operator ops.Operator) (*ops.Site, *ops.SiteO
 		return &cluster, operation, nil
 	}
 
-	// unless installing via UI, do not join until there's at least one other
-	// agent, this way we can make sure that the agent on the installer node
-	// (the one that runs "install") joins first
-	if p.OperationID == "" { // operation ID is given in UI usecase
-		report, err := operator.GetSiteInstallOperationAgentReport(operation.Key())
-		if err != nil {
-			return nil, nil, trace.Wrap(err)
-		}
-		if len(report.Servers) == 0 {
-			return nil, nil, trace.NotFound("no other agents joined yet")
-		}
-	}
+	// FIXME: this is not friends with the option when the node running the installer
```

Comment:
I failed to find a satisfactory solution for this but it prevents the CLI installer from excluding itself from the cluster (i.e. when installer uses local host as implicit future cluster node). @r0mant if you have a better idea - let's discuss.
On the flip side, I don't believe this to be much of a problem if we ignore this. If there's an agent on another node joining at the same time as the single node installer - it is user's problem (since if even tokens are the same, **it is** joining legitimately).